src/elements/content-sharing/stories/ContentSharing.stories.js

```diff
@@ -1,97 +1,32 @@
-                    language="en"
-                    token={token}
-                />
-            </IntlProvider>
-        </>
-    );
-};
-
-export const withModalAndCustomButton = () => {
-    const apiHost = text('API Host', DEFAULT_HOSTNAME_API);
-    const itemID = text('Item ID');
-    const itemType = select('Item Type', [TYPE_FILE, TYPE_FOLDER], TYPE_FILE);
-    const token = text('Access Token');
-    const customButton = <Button>&#10047; Launch ContentSharing &#10047;</Button>;
-    return (
-        <>
-            <p>
-                Update the values in the Knobs section below to view the ContentSharing UI Element. The API will be
-                instantiated on load, but the internal SharingModal will not be instantiated until the button is
-                clicked.
-            </p>
-            <IntlProvider locale="en">
-                <ContentSharing
-                    apiHost={apiHost}
-                    config={{ showEmailSharedLinkForm: false, showInviteCollaboratorMessageSection: false }}
-                    customButton={customButton}
-                    displayInModal
-                    itemID={itemID}
-                    itemType={itemType}
-                    language="en"
-                    token={token}
-                />
-            </IntlProvider>
-        </>
-    );
-};
+export const Basic = {};
 
-export const withoutModal = () => {
-    const apiHost = text('API Host', DEFAULT_HOSTNAME_API);
-    const itemID = text('Item ID');
-    const itemType = select('Item Type', [TYPE_FILE, TYPE_FOLDER], TYPE_FILE);
-    const token = text('Access Token');
-    return (
-        <>
-            <p>
-                Update the values in the Knobs section below to view the ContentSharing UI Element. The internal
-                SharingModal will appear as a form within this page when valid values are entered.
-            </p>
-            <IntlProvider locale="en">
-                <ContentSharing
-                    apiHost={apiHost}
-                    config={{ showEmailSharedLinkForm: false, showInviteCollaboratorMessageSection: false }}
-                    displayInModal={false}
-                    itemID={itemID}
-                    itemType={itemType}
-                    language="en"
-                    token={token}
-                />
-            </IntlProvider>
-        </>
-    );
+export const WithCustomButton = {
+    args: {
+        displayInModal: true,
+        customButton: <Button>&#10047; Launch ContentSharing &#10047;</Button>,
+    },
 };
 
 export default {
-    title: 'Elements|ContentSharing',
+    title: 'Elements/ContentSharing',
     component: ContentSharing,
-    parameters: {
-        notes,
+    args: {
+        apiHost: DEFAULT_HOSTNAME_API,
+        config: { showEmailSharedLinkForm: false, showInviteCollaboratorMessageSection: false },
+        displayInModal: false,
+        itemType: TYPE_FILE,
+        language: 'en',
```

Comment:
why does `ContentSharing` have a `language` arg but the others don't? is it needed or should we add to the others?